docs/DEVNOTES.md

```diff
@@ -39,12 +39,15 @@ implementation should be ever get there (react/angular/etc.).
 * Grails 3.3.8: https://grails.org/download.html
 * Gradle 3.5
 * Groovy 2.4.11
+* Liquibase 3.5.5
```

Comment:
I don't think you need this line here. This is meant to be a list of things the developer might need to install.